src/yb/tools/yb-admin-test.cc

```diff
@@ -538,6 +538,21 @@ TEST_F(AdminCliTest, TestGetClusterLoadBalancerState) {
   ASSERT_NE(output.find("ENABLED"), std::string::npos);
 }
 
+TEST_F(AdminCliTest, TestModifyPlacementPolicy) {
+  BuildAndStart();
+
+  // Modify the cluster placement policy to consist of 2 zones.
+  ASSERT_OK(CallAdmin("modify_placement_info", "c.r.z0,c.r.z1:2,c.r.z0:2", 5, ""));
+
+  auto output = ASSERT_RESULT(CallAdmin("get_universe_config"));
+
+  std::string expected_placement_blocks = "[{\"cloudInfo\":{\"placementCloud\":\"c\",\"placementRegion\":\"r\","
```

Comment:
This line is > 100 characters. Our code style is that we generally keep lines within 100 characters. Hence break it down.